### DISCUSSION TITLE
attrs[':path'].startswith('/test')

### DISCUSSION THREAD

**Discussion by ilee-y** [v2fly/v2ray-core#2494] [Q&A]
attrs: string

（V2Ray 4.18+）一段脚本，用于检测流量的属性值。当此脚本返回真值时，此规则生效。

脚本语言为 Starlark，它的语法是 Python 的子集。脚本接受一个全局变量 attrs，其中包含了流量相关的属性。

目前只有 HTTP 入站代理会设置这一属性。

有什么办法能实现这个目的，在HTTPS的时候？

**Comment by mydogshitgold** [comment]
I don't think that's possible without doing MITM.

**Reply by ilee-y** [reply]
You are right, I added another layer and tried using mitmproxy. But, I encountered a new problem, haha.
